cmd/motion/main.go

```diff
@@ -27,12 +27,29 @@ func main() {
 				Value:       os.TempDir(),
 				EnvVars:     []string{"MOTION_STORE_DIR"},
 			},
+			&cli.BoolFlag{
+				Name:        "experimentalSingularityStore",
+				Usage:       "whether to use experimental Singularity store as the storage and deal making engine",
+				DefaultText: "Local storage is used",
+			},
 		},
 		Action: func(cctx *cli.Context) error {
 			storeDir := cctx.String("storeDir")
-			logger.Infow("Using local blob store", "storeDir", storeDir)
+			var store blob.Store
+			if cctx.Bool("experimentalSingularityStore") {
+				var err error
+				store, err = blob.OpenSingularityStore(storeDir)
+				if err != nil {
+					return err
+				}
+				logger.Infow("Using Singularity blob store", "storeDir", storeDir)
```

Comment:
Add the `.Start` call here and keep the blob.Store interface small?

Reply:
Agree fine with that.